src/Widgets/WorkspaceClone.vala

```diff
@@ -44,19 +44,36 @@ namespace Gala {
         public override void paint (Clutter.PaintContext context) {
             base.paint (context);
 
-            unowned Cogl.Framebuffer fb = context.get_framebuffer ();
-
-            pipeline.set_color4ub (0, 0, 0, 100);
-            fb.push_rectangle_clip (0, 0, width, height);
-            fb.draw_rectangle (pipeline, 0, 0, width, height);
-            fb.pop_clip ();
+            var surface = new Cairo.ImageSurface (Cairo.Format.ARGB32, (int) width, (int) height);
```

Comment:
I have no idea if it is a better thing, but GNOME Shell is using Cairo.Format.A8 to draw its shadow so that it is color agnostic

Reply:
Hm I'm getting some weird artifacts when trying that